docs/batch_inference_with_mms.md

```diff
@@ -0,0 +1,335 @@
+# Batch Inference with Model Server
+
+## Contents of this Document
+* [Introduction](#introduction)
+* [Batching example](#batch-inference-with-mms-using-resnet-152-model)
+  * [Model Handler Code](#model-handler-a.k.a.-entry-point)
+    * [Initialization logic](#initialization-logic)
+    * [Preprocess logic](#preprocess-logic)
+    * [Inference logic](#inference-logic)
+    * [Postprocess logic](#postprocess-logic)
+  * [MMS Configuration](#mms-configuration)
+* [Demo](#demo-to-configure-mms-with-batch-supported-model)
+  * [Prerequisites](#pre-requisites)
+  * [Running MMS with batch inference](#loading-resnet-152-which-handles-batch-inferences)
+* [Performance results](#performance-benchmarking)
+* [Conclusion](#conclusion)   
+
+## Introduction
```

Comment:
We'd better mention trade off between throughput and latency.

Reply:
I had done performance benchmarking and explained this in that section and the conclusion section. Could you let me know what else I can add?